### PR TITLE
add genie

### PR DESCRIPTION
**Pull request by sakkayaphab**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/genie/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{% set version = "0.4" %}
+
+package:
+  name: genie
+  version: {{ version }}
+
+source:
+  url: https://github.com/sakkayaphab/genie/releases/download/v{{ version }}/genie_darwin_amd64.tar.gz # [osx]
```
Comment (bgruening) [inline recipes/genie/meta.yaml]: Can you please try to compile this tool?

File: recipes/genie/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  md5: d395c491ebb4f5a68140537e551a2b26 # [osx]
+
+  url: https://github.com/sakkayaphab/genie/releases/download/v{{ version }}/genie_linux_amd64.tar.gz # [linux]
+  md5: ab91cf7937fca654d56d9135585534ed # [linux]
+
+build:
+  number: 0
+  skip: False
```
Comment (bgruening) [inline recipes/genie/meta.yaml]: this skip should be removed

File: recipes/genie/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+test:
+  commands:
+    - genie -h
+
+about:
+  home: https://github.com/sakkayaphab/genie
+  summary: Genie is a bioinformatics tools for managing high-throughput sequencing data
```
Comment (bgruening) [inline recipes/genie/meta.yaml]: please add a library_file

File: recipes/genie/meta.yaml (hunk 1 above)
Comment (sakkayaphab) [reply recipes/genie/meta.yaml]: I've recompiled

File: recipes/genie/meta.yaml (hunk 2 above)
Comment (sakkayaphab) [reply recipes/genie/meta.yaml]: I've removed

File: recipes/genie/meta.yaml (hunk 3 above)
Comment (sakkayaphab) [reply recipes/genie/meta.yaml]: I've added

File: recipes/genie/meta.yaml (hunk 1 above)
Comment (dpryan79) [reply recipes/genie/meta.yaml]: Björn meant compile it in the recipe, rather than packaging a binary.

Comment (sakkayaphab) [reply recipes/genie/meta.yaml]: Sorry i misunderstood, but now I have fixed

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**